src/stdbuf/stdbuf.rs

```diff
@@ -1,5 +1,4 @@
 #![crate_name = "stdbuf"]
-#![feature(negate_unsigned)]
```

Comment:
No the change is fine. I was just rambling about general usefulness of this specific util.